src/core/connectors.js

```diff
@@ -2317,6 +2317,15 @@ const connectors = [{
 	],
 	js: 'connectors/uwu-radio.js',
 	id: 'uwu-radio',
+}, {
+	label: 'MyStreamPlayer',
```

Comment:
Would it be better to make this target specifically bagel radio using the mystreamplayer connector? 

I feel that would be nicer from a user perspective and then have a standalone mystreamone if needed.

cc @hornbuckle as you raised this kinda configuration before.

Reply:
@inverse That seems reasonable, then it would have Bagel Radio by name in the supported sites too.

Reply:
Agreed. Similar to what was done for #3474 and #3477 (and even earlier on #2040).

Reply:
Just split those off into separate entries